CHANGES.md

```diff
@@ -1,5 +1,152 @@
 # Changes
 
+## stable-2.12.0
+
+This release introduces route-based policy to Linkerd, allowing users to define
+and enforce authorization policies based on HTTP routes in a fully zero-trust
+way. These policies are built on Linkerd's strong workload identities, secured
+by mutual TLS, and configured using types from the Kubernetes [Gateway API](https://gateway-api.sigs.k8s.io/).
+
+The 2.12 release also introduces access logging (an optional feature that
+enables Linkerd to produce Apache-style request logs), optional support for
+`iptables-nft`, and a host of other improvements and performance
+enhancements.
+
+Additionally, the installation process has been updated to separate management
+of the Linkerd CRDs from the main installation process. With the CLI, you'll
+need to `linkerd install --crds` before running `linkerd install`; with Helm,
+you'll install the new `linkerd-crds` chart, then the `linkerd-control-plane`
+chart. These charts are now versioned using [SemVer](https://semver.org)
+independently of Linkerd releases. For more information, see the
+[upgrade notes][upgrade-2120].
+
```

Comment:
in the 2.11 changelog, we mention that the SMI extension will be required to use TrafficSplit in 2.12, but we don't mention that anywhere in the 2.12 changelog as far as i can tell. is that still the case, and if so, should we mention it here as well?